.github/workflows/clp-pr-title-checks.yaml

```
@@ -0,0 +1,23 @@
+name: "clp-pr-title-checks"
+
+on:
+  pull_request_target:
+    types: ["edited", "opened", "reopened"]
+    branches: ["main"]
+
+permissions:
+  pull-requests: "read"
```

I just realized this should be set at the job level, and should have a comment:

```suggestion
permissions:
  # For amannn/action-semantic-pull-request
  pull-requests: "read"
```